source/ranges.tex

```diff
@@ -3759,7 +3755,7 @@
     using @\exposid{Parent}@ =                              // \expos
       conditional_t<Const, const transform_view, transform_view>;
```

Comment:
```suggestion
      @\exposid{maybe-const}@<Const, transform_view>;
```

Reply:
This change wasn't mentioned in the paper AFAIK - open a separate issue for it?

Reply:
It follows from the request to use _`maybe-const`_`<Const, X>` instead of `conditional_t<Const, const X, X>`.

Reply:
OK, added as a separate editorial commit.